### DISCUSSION TITLE
Dylib Hijack Scanner

### DISCUSSION THREAD

**Discussion by dferdo61** [elrumo/macOS_Big_Sur_icons_replacements#1440] [Icon request]
Name: Dylib Hijack Scanner/DHS
Developer: Objective-See
Link: https://objective-see.org/products/dhs.html
Original icon:
![icon](https://user-images.githubusercontent.com/22680916/218308544-dfa5c28f-585b-4d14-864a-be1679849b3d.png)

**Comment by SoCuul** [comment]
Waiting for review at: https://macosicons.com/#/u/socuul
Hope this helps!
